moveit_core/robot_model/include/moveit/robot_model/joint_model_group.h

```diff
@@ -582,9 +582,10 @@ class JointModelGroup
   bool isValidVelocityMove(const double* from_joint_pose, const double* to_joint_pose, std::size_t array_size,
                            double dt) const;
 
-protected:
+  /** \brief Computes the indices of joints given a vector of joint names to look up */
   bool computeIKIndexBijection(const std::vector<std::string>& ik_jnames, std::vector<size_t>& joint_bijection) const;
```

Comment:
@henningkayser I switched this implementation to your suggestion and seems to work fine.

Question, though: Should we rename this function to not have a name targeted to IK, at this point?

Reply:
`computeJointVariableIndices()` SGTM.

I'm building these changes against Studio + running CI here to verify that they work, and if they do I'll do this as well.